tests/api_gov.spec.js

```diff
@@ -366,3 +366,25 @@ describe('TESTING /v2/gov/india', () => {
             });
     });
 });
+
+describe('TESTING /v2/gov/vietnam', () => {
```

Comment:
Consider adding another test to test that the data returned doesn't just have properties set, but has then set correctly. For example, making sure cases >= 0.

Reply:
we should do that for all gov tests yeah, will do